### PR TITLE
revised

### PR DESCRIPTION
**Pull request by honeybadgeralex**
1st iteration

### CODE REVIEW

File: content/_getting-started/first-flow.md [hunk 1]
```diff
@@ -1,172 +1,123 @@
...
-to synchronize the data between two applications or services.
+This document provides a step-by-step instruction on creating a basic [Integration Flow](integration-flow). It is intended for platform users who have little to no experience in Flow creation. The instruction includes the following sections: [configuring trigger](#configuring-trigger), [configuring action steps](#configuring-action-steps) and [post-creation activities](#post-creation-activities).
 
-We assume that this is your first experience with the {{site.data.tenant.name}}
-platform, so we explain every step in detail. Please follow these steps to create
-your first integration flow.
+## Configuring Trigger
 
-## Choosing the data source
+The first step of an Integration Flow is called an initial trigger. Its job is to trigger the running Flow execution. [Components](/integration-component) that have a trigger mode can be used for a trigger, for example: Node.js, Webhook Component, Simple Trigger, etc. For the first step, the list of available components is limited to those that have trigger mode. In this instruction we will use Simple Trigger as an example of the first step.
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: > ... that have a trigger mode can ....

We never use the word mode so far. Maybe bette just?

> that have a trigger can

Comment (drobiazko) [reply content/_getting-started/first-flow.md]: Same everywhere where you use `trigger mode`

File: content/_getting-started/first-flow.md [hunk 2]
```diff
@@ -1,172 +1,123 @@
...
 
-![Create the credentials to access](/assets/img/getting-started/first-flow/getting-started-flow-003.png "Create the credentials to access")
+The platform prompts you to setup the first step. First you may give the step a special name. Then you can select the Component you want to use, by using search bar, Component list, or load new Component from Catalog:
 
-Next, you will see the page *Choose Credentials* where you need to click on
-*+connect new credentials* link to create your first credentials. Let us fill-in
-an input form for these two fields:
-*   **Account name** - you can leave this unchanged or provide your own.
-*   **API key** - please use `{{site.data.tenant.petStoreApiKey}}` to get access to our [Petstore API]({{site.data.tenant.petStoreDocs}}).
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_5.png)
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: This screenshot contains a possibility to `Load new Component from Catalog` which is a custom implementation of one of our customers. Our product is not delivered with that option. Please create a new screenshot.

File: content/_getting-started/first-flow.md [hunk 3]
```diff
@@ -1,172 +1,123 @@
...
 
-![Select our Petstore component](/assets/img/getting-started/first-flow/getting-started-flow-002.png "Select our Petstore component")
+This brings you to *Designer* page. Let's start with naming your Flow, which is optional, and adding the initial trigger:
 
-For that purpose type `petstore` in the search bar and you will find it. Next,
-click on *Choose Petstore API (Node.js)* button.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_4.png)
 
-![Create the credentials to access](/assets/img/getting-started/first-flow/getting-started-flow-003.png "Create the credentials to access")
+The platform prompts you to setup the first step. First you may give the step a special name. Then you can select the Component you want to use, by using search bar, Component list, or load new Component from Catalog:
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: While reading
> First you may give the step a special name

I was not sure what screenshot are you talking above. I was assuming you are talking about the following screenshot but you are talking about the screenshot above. Maybe we should explicitly mention something like this

> In the screenshot above you can see ....

Plus we could place some number on the arrows in the screenshots and use the same numbers in the text. For example:

> The platform prompts you to setup the first step. First you may give the step a special name (1). Then ...

Comment (drobiazko) [reply content/_getting-started/first-flow.md]: Also see my comment below regarding `load new Component from Catalog`

File: content/_getting-started/first-flow.md [hunk 4]
```diff
@@ -1,172 +1,123 @@
...
-*+connect new credentials* link to create your first credentials. Let us fill-in
-an input form for these two fields:
-*   **Account name** - you can leave this unchanged or provide your own.
-*   **API key** - please use `{{site.data.tenant.petStoreApiKey}}` to get access to our [Petstore API]({{site.data.tenant.petStoreDocs}}).
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_5.png)
 
-Click on *Save* to verify the credentials. The verification can take no more than
-30 seconds. Upon completion you will see the page to choose one of the 3 triggers
-available in the Petstore component.
+We want to use Simple Trigger, so we will start typing "simple" to the search bar, and find it faster. Then select it and click "Choose Simple trigger":
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: Search field, not bar?

File: content/_getting-started/first-flow.md [hunk 5]
```diff
@@ -1,172 +1,123 @@
...
-![Select the trigger](/assets/img/getting-started/first-flow/getting-started-flow-004.png "Select the trigger")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_6.png)
 
-Typically a component can have more than one triggers to retrieve different
-objects from the source API. Here we use the Petstore component to learn the
-{{site.data.tenant.name}} platform. It provides 3 different triggers to retrieve
-pets by their status. These 3 triggers differ by the implemented code style. So
-select any of them, for example click on the *Get Pets By Status With Promises*
-radio button to continue.
+The next step is [data sample](/data-sample-overview) retrieval. Click "Retrieve sample from Simple trigger" and then "Continue" to proceed:
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: Maybe we should mention here that we don't want to discuss samples here as it goes beyond simple introduction? Also a link to sample docs for more details would help

File: content/_getting-started/first-flow.md [hunk 6]
```diff
@@ -1,172 +1,123 @@
...
 
-![Choose trigger parameter](/assets/img/getting-started/first-flow/getting-started-flow-005.png "Choose trigger parameter")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_7.png)
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: Between two screenshots there must be some text explaining the difference or how you got from one to another. It just looks a bit ugly if you place two screenshots one below another

File: content/_getting-started/first-flow.md [hunk 7]
```diff
@@ -1,172 +1,123 @@
...
-![Continue to finish step1 configuration](/assets/img/getting-started/first-flow/getting-started-flow-008.png "Continue to finish step1 configuration")
+As our initial trigger is ready, let's proceed with the second step, which is an action triggered by the first step. To start configuring the action, click "Add a new action":
 
-Please click on Continue button here to go further.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_10.png)
 
-> **Note**: Please don't click on *Retrieve sample from Petstore API (Node.js)*
-> button again. You will get back to the earlier stage to select the sample again.
-> This is not a problem, but unnecessary at this stage.
+Again, we're prompted to setup the step, similar to the first one. We have chosen Node.js as the second Component. Select it and click "Choose Node.js Code":
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: Phew! Node.js in the article about a Basic integration Flow? We assume the readers of the platform are beginners and Node.js is way to heavy for them. Let's choose another action

File: content/_getting-started/first-flow.md [hunk 8]
```diff
@@ -1,172 +1,123 @@
...
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_17.png)
 
-![Retrieve a sample from Email](/assets/img/getting-started/first-flow/getting-started-flow-012.png "Retrieve a sample from Email")
+Email Component sends an email when triggered, so you should just fill in the details as you would for an email. Also, you can cancel error message on the event of sending fail. After you've entered all the data, click "Continue":
 
-On the next page click on *Retrieve sample from E-Mail* to retrieve the sample.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_18.png)
 
-![Continue with the Email components' sample](/assets/img/getting-started/first-flow/getting-started-flow-013.png "Continue with the Email components' sample")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_19.png)
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: Maybe a few words about the mapping results? Why do I see `RRRAAAWWW`? I guess we need to explain samples above a little bit or mention here that the user will understand it by reading another article linked.

File: content/_getting-started/first-flow.md [hunk 9]
```diff
@@ -1,172 +1,123 @@
...
 
-![Publish the draft](/assets/img/getting-started/first-flow/getting-started-flow-016.png "Publish the draft")
 
-Before the flow can be used we need to publish the current draft. After you
-publish this flow you cannot change this version of the flow, unless you create a
-[new draft of this flow](/integrator-guide/managing-flow-history). Click on *Publish Draft*
-button to finish modifying it and saving this version of the flow.
 
-## Running and monitoring the flow
+## Post-Creation Activities
```
Comment (drobiazko) [inline content/_getting-started/first-flow.md]: I think we should move this into separate articles. The topics covered here are too heavy for beginners. Maybe we should just tell the user how to start the flow, how go to Dashboard and see the execution results. What do you think?

File: content/_getting-started/first-flow.md (hunk 5 above)
Comment (honeybadgeralex) [reply content/_getting-started/first-flow.md]: Line 36 has a link.

File: content/_getting-started/first-flow.md (hunk 4 above)
Comment (honeybadgeralex) [reply content/_getting-started/first-flow.md]: No difference )

File: content/_getting-started/first-flow.md (hunk 3 above)
Comment (honeybadgeralex) [reply content/_getting-started/first-flow.md]: This is strange. I was talking about the screenshot below.

Also, I'm a bit prejudiced to text/numbers in screenshots. I'm going to make a split-test among our guys and see what they think about it.

**Comment by honeybadgeralex** [comment]
Some made. New screenshots are in the same PR, so they may show differently right now. Should be ok when merged.

Going to ask the guys about numbering on screenshots.

**Comment by honeybadgeralex** [comment]
Alright, our office is almost unanimous for numbering. Gonna add numbers on every screenshot we have.

**Comment by drobiazko** [comment]
One minor comment. It looks like there is a typo:

> is limited to can be a trigger,

**Pull request merged**